tests/refinements/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -2466,8 +2465,7 @@ type Breakfast = Apple | Orange | Broccoli | Carrot;
 function bar(x: Breakfast) {
   if (x.kind === \"Fruit\") {
     (x.taste: \"Good\");
-  } // error, Apple.taste = Bad else
-    (x.raw: \"No\"); // error, Carrot.raw = Maybe
+  } // error, Apple.taste = Bad else (x.raw: \"No\"); // error, Carrot.raw = Maybe
```

Comment:
Yeah, this doesn't have anything to do with your pull request, it was already broken. I just wanted to mention it as it's now broken in a different way :p